juice-containers.yml

```diff
@@ -64,6 +64,25 @@ resources:
     password: ((quay-password))
 
 
+- name: container-pureos-default
+  type: docker-image
+  source:
+    repository: hub.docker.com/r/subversiveowl/pureos8-base
+
+- name: container-pureos-native
+  type: docker-image
+  source:
+    repository: hub.docker.com/r/subversiveowl/pureos8-base
+- name: container-pureos-cuda
+  type: docker-image
+  source:
+    repository: hub.docker.com/r/subversiveowl/pureos8-base
+    
+- name: container-pureos-opencl
+  type: docker-image
+  source:
+    repository: hub.docker.com/r/subversiveowl/pureos8-base
+
```

Comment:
hope you did not create those manually :)

just update the `templates/*.yml`, the `src/main.rs` and regenerate everything via `cargo b && cargo r` - this undocumented since I am going to revamp the `yml` generation from using `askama` to (probably) `tera`.